### ISSUE TITLE
One-Click Postage & Peepr

### ISSUE THREAD

**Issue by wooziwoozi** [new-xkit/XKit#874]
This isn't that big of a deal, but I use peepr a LOT with blog tracker (since view on dash is forever broken) and the tags field doesn't clear between reblogs. It still works, but I have to manually delete the last tags I typed between posts.

Like I said, it isn't a big deal, but as it is broken, and I didn't know if anyone was aware of this issue, I thought I would mention it!!

**Comment by billysanca** [comment]
@wooziwoozi how do you do to use peepr and blog tracker together? Because when I disable View on Dash and click on a tracked blog, a new tab appears!

**Comment by wooziwoozi** [comment]
@billysanca  I click on the username in the Blog Tracker sidebar (http://i.imgur.com/Cjn8P05.png), and it opens View on Dash, and then i click on the username (http://i.imgur.com/wUOSApG.png), and it opens Peepr (http://i.imgur.com/DcwWIt7.png). 

It would be nice to have an option to skip View on Dash completely, as it is broken.

Also, the tags still do not clear in Peepr.

**Comment by billysanca** [comment]
> It would be nice to have an option to skip View on Dash completely, as it is broken.

THIS!

**Comment by billysanca** [comment]
> It would be nice to have an option to skip View on Dash completely, as it is broken.
THIS ^

and thanks for answer me

**Comment by nightpool** [comment]
Can't you just uninstall View on Dash if it doesn't work for you?
@billysanca

**Comment by billysanca** [comment]
@nightpool I did it and then when I click on some tracked blog, instead of open in Peppr, it opens in a new tab.

**Comment by wooziwoozi** [comment]
@nightpool [My old way of getting peeper to work](https://github.com/new-xkit/XKit/issues/874#issuecomment-183062377) is not working at all now. It would be really nice if we could have an option for blog tracker to open in Peepr.

**Comment by ThePsionic** [comment]
@wooziwoozi I kinda made #1137 for that purpose, but there seems to be significant enough debate over whether it's worth it to turn VoD into a glorified peepr opener.

**Comment by nightpool** [comment]
After #1163, Blog Tracker will use Peepr instead of View On Dash by
default. I'm down to change other things too, if they're also using them.

**Comment by wooziwoozi** [comment]
You are my favorite person @nightpool.

**Comment by nightpool** [comment]
I wish I could auto-collapse parts of an issue discussion. 

Medium because OCP is very widely used, but not High because its not a huge bug

**Comment by ThePsionic** [comment]
Bump - this issue still exists and kind of makes OCP a PITA to use in the peepr.

**Comment by ThePsionic** [comment]
From support (`Fingered Witch#1445`)
> Is there any possibility you might do something like add a little 'clear tags' button for One Click Reblog? My tags from the previous post often carryover when I'm reblogging from mobile view, and it results in a lot of frustrating and misplaced tags!!
